features/telemetry/protocols/bmp/transport/src/main/resources/transport.proto

```diff
@@ -68,6 +68,7 @@ message PeerUpPacket {
         fixed32 as = 2;
         uint32 hold_time = 3;
         IpAddress id = 4;
+        string capabilities = 5;
```

Comment:
Can we carry the capability structures over and only convert to strings in the BMP forwarder?

This will give us more flexibility to do things with them later.

Reply:
Ok. I'll change it.

Reply:
done